include/api.php

```diff
@@ -3564,7 +3565,7 @@ function api_friendica_group_delete($type) {
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.